packages/nextjs/app/admin/_components/GrantReview.tsx

```diff
@@ -119,8 +147,17 @@ export const GrantReview = ({ grant, selected, toggleSelection }: GrantReviewPro
             )}
           </div>
         </div>
-        <div className="flex gap-4 items-center">
+        <div className="flex gap-4 items-center relative">
           <Address address={grant.builder} link={`https://app.buidlguidl.com/builders/${grant.builder}`} />
+          {grantsCount > 0 && (
+            <span className="group text-sm text-gray-500">
+              {grantsCount} {grantsCount === 1 ? "submission" : "submissions"}{" "}
+              <QuestionMarkCircleIcon className="h-4 w-4 inline" />
+              <span className="hidden group-hover:block absolute bg-gray-200 text-xs rounded p-1 -mt-12 ml-8">
+                {tooltipContent}
+              </span>
+            </span>
```

Comment:
nitpick: We could have used daisyUI [tooltip](https://daisyui.com/components/tooltip/), but even this approach look fine 🙌

Reply:
Ups! Sorry. Updated!